Mathlib/Analysis/InnerProductSpace/Calculus.lean

```diff
@@ -245,6 +245,37 @@ theorem DifferentiableAt.norm (hf : DifferentiableAt ℝ f x) (h0 : f x ≠ 0) :
   ((contDiffAt_norm 𝕜 h0).differentiableAt le_rfl).comp x hf
 #align differentiable_at.norm DifferentiableAt.norm
 
+theorem not_differentiableAt_abs_zero : ¬ DifferentiableAt ℝ (fun x : ℝ ↦ abs x) 0 := by
```

Comment:
```suggestion
theorem not_differentiableAt_abs_zero : ¬ DifferentiableAt ℝ (abs : ℝ → ℝ) 0 := by
```
I think its probably better to prove this about the actual function, rather than an anonymous function that does the same thing (these are basically the same but subtly different in lean)